src/components/Design/CardWrapper/PageLayout.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import Header from '../../Header/Header'
+
+export default function PageLayout({ title, children }) {
```

Comment:
Is there a need for this PageLayout component? I would move the Header title logic to your header component and your grid to your globalstyles